### PR TITLE
refactor: typescript

### PR DESCRIPTION
**Pull request by GoldElysium**

### CODE REVIEW

File: Backend/App.ts [hunk 1]
```diff
@@ -68,16 +71,12 @@ require('./auth/strategies/Discord').DiscordAuth(passport);
 app.use('/auth', AuthRouter);
 app.use('/user', UserRouter);
 
-app.get('/', async (req, res) => {
+app.get('/', async (_: express.Request, res: express.Response) => {
```
Comment (GoldElysium) [inline Backend/App.ts:74]: For whatever reason, it didn't automatically type correctly, at least in my IDE 🤷‍♂️

**Comment by GoldElysium** [comment]
Also need to go over all the TODO comments

**Comment by omneex** [comment]
changed the branch for merge to v1.3-typescript-frontend

**Comment by GoldElysium** [comment]
Rebase to solve merge comflicts

**Pull request merged**